elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemParserCommunity18_0.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.wildfly.extension.elytron;
+
+import org.jboss.as.controller.PersistentResourceXMLDescription;
+
+public class ElytronSubsystemParserCommunity18_0 extends ElytronSubsystemParser18_0 {
```

Comment:
I strongly discourage the use of polymorphism to model versioning.

Reply:
@pferraro I removed the polymorphism from versioning,  Thank you!